### PR TITLE
Document System.Reflection.Metadata.ImageFormatLimitationException

### PR DESCRIPTION
**Pull request by carlossanlop**
Documenting the 3.0 serialization API added recently by @joperezr.
Added also a general summary. There were no comments in the source, and the exception is older than .NET Core, so I based my summary on the way it's being used in corefx. The area owners are @tmat, @nguerrera. Can you please help make sure the summary is correct?

### CODE REVIEW

File: xml/System.Reflection.Metadata/ImageFormatLimitationException.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>The exception that is thrown when the heap size limit is exceeded.</summary>
```
Comment (tmat) [inline xml/System.Reflection.Metadata/ImageFormatLimitationException.xml]: I'd say more generally, that the exception is thrown when an attempt to write metadata exceeds a limit given by the format specification. Heap size limit is one of the limits.

Comment (carlossanlop) [reply xml/System.Reflection.Metadata/ImageFormatLimitationException.xml]: Ah great, thanks for the clarification. Does this sound good?:

```suggestion
    <summary>The exception that is thrown when an attempt to write metadata exceeds a limit given by the format specification. For example, when the heap size limit is exceeded.</summary>
```

Comment (carlossanlop) [reply xml/System.Reflection.Metadata/ImageFormatLimitationException.xml]: I can remove the example at the end if you don't feel like it should be included.

Comment (carlossanlop) [reply xml/System.Reflection.Metadata/ImageFormatLimitationException.xml]: @tmat I'll merge the above suggestion based on your comment, but feel free to let us know if you'd like us to change it.

**Pull request merged**